web/vtadmin/src/index.css

```diff
@@ -216,3 +216,18 @@ table tbody td {
 table tbody td[rowSpan] {
     border-right: solid 1px var(--tableBorderColor);
 }
+
+.Toastify__toast {
```

Comment:
Toastify has some styling issues; must override some of their native styles. Let me know if this should go elsewhere.